core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala

```diff
@@ -118,7 +125,9 @@ private[ui] class AllJobsPage(parent: JobsTab, store: AppStatusStore) extends We
   private def makeExecutorEvent(executors: Seq[v1.ExecutorSummary]):
       Seq[String] = {
     val events = ListBuffer[String]()
-    executors.foreach { e =>
+    executors.sortBy { e =>
+      -math.max(e.addTime.getTime, e.removeTime.map(_.getTime).getOrElse(-1L))
```

Comment:
shall we do it in the same way by sorting with both the attributes as done in case of jobs and stages.

Reply:
I don't think it's good for executors. If we do so, removed executor can disappear from the view even if the removal event is the most latest event.